### PR TITLE
database schema updated

### PR DESCRIPTION
**Pull request by itsTony4dev**
modify user `phone_number` to be unique, make job `is_approved` false by default and refactor event table by adding `eventLocationtype` table and change `type_id` to `event_location_type` in the `event` table

### CODE REVIEW

**Comment by Hkataya** [comment]
@itsTony4dev Why did you separate the event location type ?

**Comment by itsTony4dev** [comment]
@Hkataya typeId was refered to jobtype and this is not quite good so we decide to make a new table for the event type and not use jobtype as fk. Btw we spoke about it a week ago on frontend channel and if u want we can discuss it in the meeting today

**Comment by Hkataya** [comment]
@itsTony4dev do we still need this?

File: docs/db_model.dbml [hunk 1]
```diff
@@ -1,210 +0,0 @@
-/*========== USER ENTITY =============*/
```
Comment (Hkataya) [inline docs/db_model.dbml:1]: Re create file

File: docs/db_model.png
Comment (Hkataya) [inline docs/db_model.png:1]: Re create file

File: packages/backend/src/db/migrations/0001_oval_snowbird.sql
Comment (Hkataya) [inline packages/backend/src/db/migrations/0001_oval_snowbird.sql:1]: One migration file

**Pull request closed without merging**